mythril/src/error.rs

```diff
@@ -52,11 +52,12 @@ pub fn check_vm_insruction(rflags: u64, error: String) -> Result<()> {
     } else if rflags.contains(RFlags::FLAGS_ZF) {
         let errno = unsafe {
             let value: u64;
-            llvm_asm!("vmread %rax, %rdx;"
-                      : "={rdx}"(value)
-                      : "{rax}"(vmcs::VmcsField::VmInstructionError as u64)
-                      : "rflags"
-                      : "volatile");
+            asm!(
+                "vmread rdx, rax",
+                in("rax") vmcs::VmcsField::VmInstructionError as u64,
+                out("rdx") value,
```

Comment:
Doesn't this need something like `lateout("rflags") _` to ensure that the compile is aware that rflags has been clobbered by this asm?